tensorflow/python/ops/signal/helper.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Shift the zero-frequency component to the center of the spectrum"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python import roll as _roll
+from tensorflow.python.framework import ops
+from tensorflow.python.util.tf_export import tf_export
+
+@tf_export("signal.fftshift")
+def fftshift(x, axes=None):
+    """
+    Shift the zero-frequency component to the center of the spectrum.
```

Comment:
Please format the docstrings according to the style guide: 
https://www.tensorflow.org/community/contribute/code_style#python_operations

Reply:
changes done